Spanner/src/V1/SpannerClient.php

```diff
@@ -17,36 +17,18 @@
 
 /*
  * GENERATED CODE WARNING
- * This file was generated from the file
- * https://github.com/google/googleapis/blob/master/google/spanner/v1/spanner.proto
- * and updates to that file get reflected here through a refresh process.
- *
- * EXPERIMENTAL: This client library class has not yet been declared GA (1.0). This means that
- * even though we intend the surface to be stable, we may make backwards incompatible changes
- * if necessary.
- *
- * @experimental
+ * Generated by gapic-generator-php from the file
+ * https://github.com/googleapis/googleapis/blob/master/google/spanner/v1/spanner.proto
+ * Updates to the above are reflected here through a refresh process.
  */
 
 namespace Google\Cloud\Spanner\V1;
 
-use Google\ApiCore\Transport\TransportInterface;
 use Google\Cloud\Spanner\V1\Gapic\SpannerGapicClient;
 
-/**
- * {@inheritdoc}
- */
+/** {@inheritdoc} */
 class SpannerClient extends SpannerGapicClient
 {
-    /**
-     * Returns the underlying transport.
-     *
-     * @access private
-     * @return TransportInterface
-     * @experimental
-     */
-    public function getTransport()
-    {
-        return parent::getTransport();
-    }
+    // This class is intentionally empty, and is intended to hold manual additions to
+    // the generated {@see SpannerGapicClient} class.
```

Comment:
@tmatsuo Looks like we need to exclude the Partial Veneer class to preserver the customizations in `owlbot.py`